example/rootfs/etc/services.d/example/finish

```diff
@@ -1,9 +1,12 @@
-#!/usr/bin/execlineb -S1
+#!/usr/bin/env bashio
 # ==============================================================================
 # Take down the S6 supervision tree when example fails
 # s6-overlay docs: https://github.com/just-containers/s6-overlay
 # ==============================================================================
-if { s6-test ${1} -ne 0 }
-if { s6-test ${1} -ne 256 }
 
-s6-svscanctl -t /var/run/s6/services
+if [[ "$1" -ne 0 ]] && [[ "$1" -ne 256 ]]; then
+  bashio::log.warning "Service shutdown"
```

Comment:
Maybe change this to:

```suggestion
  bashio::log.warning "Halt add-on"
```

As its not just the service that stops, the whole container is being taken down.